doc/man7/ossl-guide-introduction.pod

```diff
@@ -17,7 +17,7 @@ Finally, it also has a set of providers that supply implementations of a broad
 set of cryptographic algorithms.
 
 OpenSSL is fully open source. Version 3.0 and above are distributed under the
-Apache v2 license.
+Apache v2 license. The codebase is mostly in C, C++, and Perl.
```

Comment:
It does not contain any C++, but does contain assembler (using perl to generate the assembler)

Reply:
Thanks. I can fix this. Any other changes I should make?

I was basing this on the GitHub repo summary in the sidebar:

> Languages

> C 75.0%
> Perl 21.9%
> C++ 1.6%
> Raku 0.5%
> Assembly 0.5%
> Shell 0.3%
> Other 0.2%

Reply:
oh, and we do actually have some C++ code in the tree (the nghttp3 demo), but its not part of the openssl library specifically

Reply:
Raku is basically next gen Perl.  It was once called Perl 6, but ended up changing so much from Perl 5 that it ended up being a very different language, hence a new name.

Reply:
(I'm quite curious to know what Github identified as Raku)